src/sidebars/favourites/favourites.less

```diff
@@ -1,11 +1,13 @@
-﻿@favourite-height: 100px;
+﻿@favourite-height: 110px;
 @favourite-padding-left: 0px;
-@favourite-padding-top: 5px;
-@favourite-padding-right: 4px;
-@favourite-padding-bottom: 10px;
+@favourite-padding-top: 10px;
+// Padding right is declared in the variables file.
```

Comment:
There isn't really a need for that, no. Perhaps a brief explanation why in a comment instead? I can't think of a name that would go welll, either.